### PR TITLE
Feature/implement accounts

### PR DESCRIPTION
**Pull request by OneCodeMan**
screen shot 2018-07-17 at 2 08 25 pm

### CODE REVIEW

File: td4m3/src/main/java/ca/td/td4m3/accounts/adapter/AccountsAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package ca.td.td4m3.accounts.adapter
+
+class AccountsAdapter {
```
Comment (TheRishabhB) [inline td4m3/src/main/java/ca/td/td4m3/accounts/adapter/AccountsAdapter.kt]: Should extend AbstractDataBindAdapter (I forget exact name)

File: td4m3/src/main/java/ca/td/td4m3/accounts/activity/AccountsActivity.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package ca.td.td4m3.accounts.activity
+
+class AccountsActivity {
```
Comment (TheRishabhB) [inline td4m3/src/main/java/ca/td/td4m3/accounts/activity/AccountsActivity.kt]: Should extend AbstractActivity

File: td4m3/src/main/java/ca/td/td4m3/accounts/view/AccountsView.kt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package ca.td.td4m3.accounts.view
+
+interface AccountsView {
```
Comment (TheRishabhB) [inline td4m3/src/main/java/ca/td/td4m3/accounts/view/AccountsView.kt]: Should be implemented in Activity or not needed.

File: td4m3/src/main/res/layout/accounts_view.xml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (TheRishabhB) [inline td4m3/src/main/res/layout/accounts_view.xml:2]: Where are you calling this?

Comment (OneCodeMan) [reply td4m3/src/main/res/layout/accounts_view.xml:2]: No where. I'm not sure where to call it. Any ideas?

Comment (TheRishabhB) [reply td4m3/src/main/res/layout/accounts_view.xml:2]: Think of a UI table view. You have to construct each row. You're constructing more than 1 row at a time, so this isn't what you would use for making a reusable component. I'll show you tomorrow how to make one.

Comment (TheRishabhB) [reply td4m3/src/main/res/layout/accounts_view.xml:2]: Also you can use linearlayout or relativelayout instead.

**Comment by TheRishabhB** [comment]
Looks good. You have merge could conflicts however.

**Comment by OneCodeMan** [comment]
screenshot_1531953376

**Pull request merged**